docs/data/data-grid/editing/CustomEditComponent.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import Box from '@mui/material/Box';
 import Rating from '@mui/material/Rating';
-import { unstable_useEnhancedEffect as useEnhancedEffect } from '@mui/utils';
+import useEnhancedEffect from '@mui/utils/useEnhancedEffect';
```

Comment:
This modification looks weird, because nothing indicates that this hook is unstable.

Not an issue in the package code, but it the docs, I'm not sure

Reply:
One could say that we should not even use `@mui/utils` in the doc since it's meant to be an internal package, but we are doing it extensively :grimacing:

Reply:
Problem solved? 😁

```suggestion
import unstable_useEnhancedEffect from '@mui/utils/useEnhancedEffect';
```

Otherwise, https://github.com/mui/material-ui/blob/a59ee1a371f488349dda6d4fd4993538aa0e1c77/docs/data/joy/customization/theme-colors/BootstrapVariantTokens.js#L108-L109